### ISSUE TITLE
[Feature Request] Add toggle for cursor/line diagnostics

### ISSUE THREAD

**Issue by mdietrich16** [rachartier/tiny-inline-diagnostic.nvim#45]
Would it be possible to add an option to change the behavior to surface line diagnostics instead of cursor diagnostics when on a line?

**Comment by rachartier** [comment]
You want to see all the errors, warnings... on the actual line, and not only where the cursor is ? Is it correct ?

**Comment by mdietrich16** [comment]
Yes! As far as I understand right now it kind of does this already, because it shows the first line diagnostic if no diagnostic is found for the cursor position, right? But if on a token with a diagnostic attached, then it shows that diagnostic. Depending on how #44 works, this is a mute point, I'll have to check and test that out.

**Comment by rachartier** [comment]
It should be good now.

You can set 
```lua
require("tiny-inline-diagnostic").setup({
		-- Show all diagnostics on the cursor line.
		show_all_diags_on_cursorline = true,
})
```
And it will display all diagnostics on your cursor line.
